truecase/__init__.py

```diff
@@ -1,5 +1,5 @@
 from functools import lru_cache
-
+from .Trainer import Trainer
```

Comment:
why did you have to import here?

Reply:
I added this in order to able to import the `Trainer` class in the same way as the `TrueCaser`. Is there another way to do it in the existing code?